pkg/controller/configmapcainjector/configmapcainjector_controller.go

```diff
@@ -0,0 +1,165 @@
+package configmapcainjector
```

Comment:
I suggest renaming the file to `controller.go` and the directory to `configmap_ca_injector` (idiomatic go). pkg will still be `configmapcainjector`.

Reply:
fixed